src/renderer/windows/settings/advanced-settings.tsx

```diff
@@ -22,6 +25,23 @@ export const AdvancedSettings: FunctionComponent = () => {
         <Row>
           <StyledCheckbox color={INPUT_COLOR} label="Enable dev tools" size="xs" {...register('isDebug')} />
         </Row>
+        <Row>
+          <Label>
+            Refresh Time (milliseconds)
+            <Slider
+              type="range"
+              min={1000}
+              max={10000}
+              step={500}
```

Comment:
Do we really need half a second precision on track refreshes?

```suggestion
              step={1}
```

Reply:
step={1} would be millisecond precision. We can go to step 1000 if you'd rather, i just picked it pretty much at random.

Reply:
actually you make a good point. I'll go make that seconds now. Yesterday brain was not very switched on.